code/src/components/Collection.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react'
+
+import API_URL from 'utils/Api'
+
+import Message from 'components/Message'
+import SentMessage from 'components/SentMessage'
+
+
+const Collection = () => {
+
+    const [fetchThought, setFetchThought] = useState([])
```

Comment:
Naming convention could be a bit more descriptive to what this is actually doing and makes sense in all places it's being used. So instead of `fetchThought`, something like this:
```javascript
const [messages, setMessages] = useState([]);